# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1465164091 25200
#      Sun Jun 05 15:01:31 2016 -0700
# Node ID f3455f8ff86d14b6cb0bbf07b3eaf6348230bc23
# Parent  f1f17f13b3b9f4f72d2cc545a7df0879ebc18ce0
Document new functions dialog and uibuttongroup

* NEWS: Mention new functions dialog and uibuttongroup.  Sort the list.
* __unimplemented__.m: Remove dialog from the list of missing functions.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -77,26 +77,28 @@ Summary of important user-visible change
  ** Dialog boxes--errordlg, helpdlg, inputdlg, listdlg, msgbox,
     questdlg, and warndlg--now exclusively use Qt for rendering.
     Java based versions have been removed.
 
  ** Other new functions added in 4.2:
 
       audioformats
       deg2rad
+      dialog
       evalc
       hash
       im2double
       localfunctions
-      odeset
+      ode45
       odeget
-      ode45
+      odeset
       padecoef
       psi
       rad2deg
+      uibuttongroup
 
  ** Deprecated functions.
 
     The following functions have been deprecated in Octave 4.2 and will
     be removed from Octave 4.6 (or whatever version is the second major
     release after 4.2):
 
       Function             | Replacement
diff --git a/scripts/help/__unimplemented__.m b/scripts/help/__unimplemented__.m
--- a/scripts/help/__unimplemented__.m
+++ b/scripts/help/__unimplemented__.m
@@ -623,17 +623,16 @@ function list = missing_functions ()
   "ddesd",
   "ddeset",
   "decic",
   "delaunayTriangulation",
   "depdir",
   "depfun",
   "details",
   "deval",
-  "dialog",
   "digraph",
   "discretize",
   "dither",
   "docsearch",
   "dragrect",
   "dynamicprops",
   "echodemo",
   "empty",
